dev-python/yte/yte-1.5.6.ebuild

```diff
@@ -20,3 +21,9 @@ RDEPEND="
 	dev-python/plac[${PYTHON_USEDEP}]
 	dev-python/pyyaml[${PYTHON_USEDEP}]
 "
+
+distutils_enable_tests pytest
+
+python_test() {
+	epytest tests.py || die "Tests failed with ${EPYTHON}"
+}
```

Comment:
`|| die` is not necessary here, `epytest` already does that.

Reply:
Fixed, and also removed `/refs/tags/` from github url (the link in *Releases* includes it, but it works without)